chapters/introduction.tex

```diff
@@ -1,11 +1,7 @@
 \section{Introduction} \label{sec:introduction}
-% Setting the scene for the project, giving it its context
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
+With technology steadily improving over the years, people are becoming more interconnected than ever and entertainment in form of applications are plentiful. Despite this, depression continues to rise among the general population. According to the World Health Organization, over 322 million cases of depressive disorder were recorded in 2017, representing a rise of around 18.4\% over the past ten years \cite{who_depression}. Not only does depression negatively impact wellbeing, it may also lead to complications such as increased fatigue, decreased motivation or even suicide. As a result, researchers have used computing technologies known as \textit{Affective Computing} that seek to identify human emotions in order to combat mental disorders \cite{ieee_affective}.
 
-The research objective of this paper is to design and implement the approach proposed by Moore, Galway and Donnelly in \cite{sohappy}. Finally, a conclusion is drawn in section \ref{sec:conclusion}.
+In \cite{sohappy}, Moore, Galway and Donnelly propose a smartphone application to be used in a future study. It is designed to encourage smiling, which has been shown to positively affect happiness and thus serves as an effective means to counteract depression. The research objective of this report is to design and implement the approach described by Moore, Galway and Donnelly as an Android application. In order to render the application suitable for different kinds of studies, the application's components are designed to be as interchangeable as possible, allowing individual adjustments to be made with ease.
 

```

Comment:
change: "described by Moore, Galway and Donnelly as an Android application." to described by the authors.... or similar

so the names not written twice.